realm/realm-library/src/objectServer/java/io/realm/SyncConfiguration.java

```diff
@@ -793,6 +812,15 @@ public Builder waitForInitialRemoteData() {
             return this;
         }
 
+        /**
+         * Setting this will open a partially synchronized Realm.
+         * @see #isPartialSync()
+         */
+        public SyncConfiguration.Builder withPartialSync() {
```

Comment:
Not sure what the other bindings call this. `.partialRealm()` feels more accurate to me, but can we revisit this when fleshing out the full API.

Reply:
`partialRealm ` is good 
Cocoa is using `isPartial` as a property of the SyncConfig https://github.com/realm/realm-cocoa/pull/5326/files?diff=unified#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR54

DotNet is using `IsPartial` https://github.com/realm/realm-dotnet/pull/1580/files#diff-bc069b1553b16b22b430e81c0e3b641cR95